chart/open-feature-operator/README.md

```diff
@@ -74,7 +74,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `controllerManager.manager.resources.requests.memory` | `64Mi` |
 | `managerConfig.controllerManagerConfigYaml` | `1` |
 | `managerConfig.replicas.health.healthProbeBindAddress` | `:8081` |
-| `managerConfig.replicas.metrics.bindAddress` | `127.0.0.1:8080` |
+| `managerConfig.replicas.metrics.bindAddress` | `0.2.29.1:8080` |
```

Comment:
This annoying this happened again. I will open a new PR to fix this and prevent it from recurring.